src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -91,48 +27,32 @@ type VMTableProps = {
 };
 
 const VMTable: React.FC<VMTableProps> = ({ data, unfilteredData, loaded, loadError, kind }) => {
-  const { t } = useTranslation('plugin__kubevirt-plugin');
+  const columns = useVirtualMachineColumes();
 
   return (
     <VirtualizedTable<K8sResourceCommon>
       data={data}
       unfilteredData={unfilteredData}
       loaded={loaded}
       loadError={loadError}
-      columns={columns(t)}
-      Row={VMRow}
+      columns={columns}
+      Row={VirtualMachineRow}
       rowData={{ kind }}
     />
   );
 };
 
-export const filters: RowFilter[] = [
-  {
-    filterGroupName: 'Status',
-    type: 'vm-status',
-    reducer: (obj) => obj?.status?.printableStatus,
-    filter: (statuses, obj) => {
-      const status = obj?.status?.printableStatus;
-      return (
-        statuses.selected?.length === 0 ||
-        statuses.selected?.includes(status) ||
-        !statuses?.all?.find((s) => s === status)
-      );
-    },
-    items: Object.keys(printableVMStatus).map((status) => ({
-      id: status,
-      title: status,
-    })),
-  },
-];
-
-const VirtualMachinesList = ({ kind }: { kind: string }) => {
+const VirtualMachinesList: React.FC<{ kind: string; namespace: string }> = ({
```

Comment:
what do you think about adding a type for those props?

Reply:
+1 changed thanks :)